prairial/parser.js

```diff
@@ -19,7 +19,7 @@ module.exports = new Parser(function analyseEC(parsedUrl, ec) {
 
   let match;
 
-  if ((match = /^\/([a-z0-9-]+)\/(index.php)?$/i.exec(path)) !== null) {
+  if ((match = /^\/(?!idref|prairialdoc)([a-z0-9-]+)\/(index.php)?$/i.exec(path)) !== null) {
```

Comment:
`([a-z0-9-]+)` already detects the URL you've just added, what was your objective in adding `(?!idref|prairialdoc)`

tip: you can run tests with the command : `$ ezpaarse-project/make test prairial`

Reply:
Adding (?!idref|prairialdoc) allows to discard lines that contain theese terms.
This had been tested like this :
```
echo "https://publications-prairial.fr/prairialdoc/index.php?id=214" | ./parser.js
{}
```
and like this :
`ezp process -H "Force-Parser: prairial" ezproxy.log --out result.csv`
As expected, urls with *prairialdoc* or *idref* are not retained in  the result.csv

Reply:
Okay, I get it.

In this case it's correct.

If you want to add your exceptions to the test file, you need to add the URL with all the fields empty.

Like this: 
`;;;;;https://publications-prairial.fr/idref/index.php?id=082121338`

You can add an example for URL with /prairialdoc/<something> and we'll be able to validate the pull request.

Reply:
Hello,
I added the example as you suggested (in an extra commit). Don't know if there is anything more to do.